pep-0526.txt

```diff
@@ -298,6 +309,12 @@ Changes to standard library and documentation
   For classes it returns ``collections.ChainMap`` constructed from
   annotations in method resolution order of that class.
 
+- The special type construct ``_ClassVar`` is added to ``typing`` to indicate
```

Comment:
Wait, I thought this was `ClassVar` (no underscore)? I mean, I was super busy and got lost in the megathread, so I might have missed something...